modules/ROOT/pages/logs.adoc

```diff
@@ -241,12 +240,12 @@ For every result group, the *Quick Filters* list displays the first five results
 
 This example of a predefined filter selects the ERROR log level.
 
-image::log-filter-select.png[Predefined Log Filter]
+image::log-filter-select.png["Predefined log filter"]
 
 After you select a predefined filter from the Quick Filter list, you see it
 near the top of the page, next to *Add a Filter*.
 
-image::log-filter-actions.png[Filter Actions]
+image::log-filter-actions.png["Filter actions"]
```

Comment:
You could give more context to these alt texts - maybe "Predefined filter actions on the xxx page"